share/nitweb/directives/entity/doc.html

```diff
@@ -1,8 +1,29 @@
 <div class='card' ng-if='mentity.mdoc'>
 	<div class='card-body'>
 		<div class='pull-right'>
-			<entity-rating mentity='mentity' />
+			<div class='dropdown'>
+				<button class='btn btn-link dropdown-toggle' type='button' data-toggle='dropdown'>
+					<span class='glyphicon glyphicon-chevron-down'></span>
+				</button>
+				<ul class='dropdown-menu dropdown-menu-right'>
+					<li ng-class='currentTab == "doc" ? "active" : ""'>
+						<a ng-click='currentTab = "doc"'>Doc</a>
+					</li>
+					<li ng-class='currentTab == "grade" ? "active" : ""'>
+						<a ng-click='currentTab = "grade"'>Grade</a>
```

Comment:
I do not know why, but *grade* make me think of beef. http://www.darcysmeats.ca/2014/09/23/understanding-canadian-beef-grades/